### DISCUSSION TITLE
error open file

### DISCUSSION THREAD

**Discussion by farshad-fallah** [BearWare/TeamTalk5#2250] [Q&A] [answered]
hi hello my server is ubuntu 2204 and wen I want creat new user I hit add then say an error open file failed
I cant creat new user 
how I can fix it thanks

**Comment by lbk2907** [comment]
Make sure you have properly set the permition to the XML file. RWX-RWX-R-x
(775) is good enough.
chmod 775 /path/to/your/xml/file

**Reply by farshad-fallah** [reply]
thanks but it dosenot work for me and prob not fix 
I changed  permition

**Comment by lbk2907** [comment]
May you contact me privately? I'll see what can I help. Go to my profile to
have my contact information.

**Comment by bear101** [comment, marked as answer]
Do you have write access to the location of your file storage? Try "touch /file/storage/location/myfile" and see if you can create a file.

**Comment by farshad-fallah** [comment]
Thank you, I saw that the space was full and I deleted the files and it was fixed
Now I can create a userI have another problem

The log is not created for me even though it is ticked
